apis/v1alpha1/types.go

```diff
@@ -76,19 +76,23 @@ type ReleaseParameters struct {
 type ReleaseObservation struct {
 	State              release.Status `json:"state,omitempty"`
 	ReleaseDescription string         `json:"releaseDescription,omitempty"`
+	Revision           int            `json:"revision,omitempty"`
 }
 
 // A ReleaseSpec defines the desired state of a Release.
 type ReleaseSpec struct {
 	runtimev1alpha1.ResourceSpec `json:",inline"`
 	ForProvider                  ReleaseParameters `json:"forProvider"`
+	RollbackLimit                *int32            `json:"rollbackLimit,omitempty"`
 }
 
 // A ReleaseStatus represents the observed state of a Release.
 type ReleaseStatus struct {
 	runtimev1alpha1.ResourceStatus `json:",inline"`
 	AtProvider                     ReleaseObservation `json:"atProvider,omitempty"`
 	PatchesSha                     string             `json:"patchesSha,omitempty"`
+	Failed                         int32              `json:"failed,omitempty"`
```

Comment:
maybe NumFailed or NumFailures or FailureCounter.
To point out it's purpose is to track number of failures, otherwise looks like a bool to me (especially next to Synced field).

Reply:
I agree that it may sound like a bool but I did follow the same convention as Kubernetes Jobs: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/batch/types.go#L185
While checking yaml, it will be displayed as 1,2,3,4..., so may be it is not a big deal. 

Considering above, do you think we should still change? I am not too opinionated just tend to follow existing conventions.